minet/buzzsumo/client.py

```diff
@@ -191,6 +190,6 @@ def domain_articles(self, domain, begin_timestamp, end_timestamp):
                     break
 
                 for article in data["results"]:
-                    yield format_article(article)
+                    yield BuzzsumoArticle.from_payload(article)
```

Comment:
There is no difference anymore between article and exact_url?

Reply:
No. I found that the data fields that Buzzsumo returns are the same for both endpoints.